packages/instant/src/components/zero_ex_instant_container.tsx

```diff
@@ -45,14 +46,16 @@ export class ZeroExInstantContainer extends React.Component<ZeroExInstantContain
                         overflow="hidden"
                         height="100%"
                     >
-                        <Flex direction="column" justify="flex-start" height="100%">
-                            <SelectedAssetInstantHeading onSelectAssetClick={this._handleSymbolClick} />
-                            <SelectedAssetBuyOrderProgress />
-                            <LatestBuyQuoteOrderDetails />
-                            <Container padding="20px" width="100%">
-                                <SelectedAssetBuyOrderStateButtons />
-                            </Container>
-                        </Flex>
+                        <LatestErrorOverlay>
```

Comment:
Can it just be a sibling?

Reply:
Yes, thanks for calling this out.  This hierarchy was a relic of when the overlay was implemented with an `:after` css selector